packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, ComponentsOverrides, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import * as React from "react";
+
+export type MenuItemGroupClassKey = "root" | "title";
+
+const styles = (theme: Theme) =>
+    createStyles<MenuItemGroupClassKey, MenuItemGroupProps>({
+        root: { marginTop: theme.spacing(8) },
+        title: {
+            fontWeight: theme.typography.fontWeightBold,
+            fontSize: 14,
+            lineHeight: "20px",
+            borderBottom: `1px solid ${theme.palette.grey[50]}`,
+            padding: theme.spacing(2, 4),
+        },
+    });
+
+export interface MenuItemGroupProps {
+    title?: string | React.ReactNode;
```

Comment:
```suggestion
    title?: React.ReactNode;
```

`string` is part of React.ReactNode already